meta-balena-common/recipes-support/hostapp-update-hooks/hostapp-update-hooks.bb

```diff
@@ -25,9 +29,10 @@ FILES_${PN} += " \
 	"
 
 RDEPENDS_${PN} = " \
-	util-linux \
-	balena \
-	"
+    balena \
+    dropbear \
+    util-linux \
+    "
```

Comment:
should this also RDEPEND on openssh so that the `ssh-keygen` utility comes?
Or does that utility also come from dropbear?

Reply:
I added the dependency.